include/signalrclient/hub_proxy.h

```diff
@@ -74,4 +61,17 @@ namespace signalr
         SIGNALRCLIENT_API pplx::task<void> __cdecl invoke_void(const utility::string_t& method_name, const web::json::value& arguments,
             const on_progress_handler& on_progress);
     };
+
+    template<>
+    inline pplx::task<void> hub_proxy::invoke<void>(const utility::string_t& method_name, const on_progress_handler& on_progress)

```

Comment:
Yes. It won't compile with gcc. From what I remember when I hit this it turned out that the compiler in VS allows both but only the definition outside the class is per standard and the definition inside class is an extension